### ISSUE TITLE
Case sensitive jquery importing

### ISSUE THREAD

**Issue by alex88** [Dogfalo/materialize#2846]
I've a warning from webpack saying there is a different case importing of jquery and I've seen it's from the materialize package.

In fact sometimes you `require('jquery')` and sometimes `require('jQuery')`. If i do a PR will you accept that? What's the preferred way? jquery or jQuery? [Here](https://www.npmjs.com/package/jQuery) they say jQuery.

**Comment by scarletsky** [comment]
You can use webpack's `resolve.alias` option to fix this.

```js
{
  entry: ...,
  output: ...,
  resolve: {
    alias: {
      'jQuery': 'jquery'
    }
  }
}
```

**Comment by Dogfalo** [comment]
Merged #2870
